GitExtensions.sln

```diff
@@ -393,8 +393,8 @@ Global
 		{2B9B62FA-3E0C-45C5-9287-68B69413C5DF}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{2B9B62FA-3E0C-45C5-9287-68B69413C5DF}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{2B9B62FA-3E0C-45C5-9287-68B69413C5DF}.Release|Any CPU.Build.0 = Release|Any CPU
-		{2D18BE89-D210-49EB-A9DD-2246FBB3DF6D}.Debug|Any CPU.ActiveCfg = Release|Any CPU
-		{2D18BE89-D210-49EB-A9DD-2246FBB3DF6D}.Debug|Any CPU.Build.0 = Release|Any CPU
+		{2D18BE89-D210-49EB-A9DD-2246FBB3DF6D}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{2D18BE89-D210-49EB-A9DD-2246FBB3DF6D}.Debug|Any CPU.Build.0 = Debug|Any CPU
```

Comment:
Please revert, I vaguely remember there was an issue using the Debug build

Reply:
Ok.